scripts/md2html/build.sh

```diff
@@ -58,9 +58,11 @@ for specification in $specifications; do
 
   node scripts/md2html/md2html.js --maintainers $maintainers $specification "$allVersions" > $tempfile
   npx respec --no-sandbox --use-local --src $tempfile --out $tempfile2
```

Comment:
Consider using the `TZ` env var while running `npx respec` to ensure more-consistent output, regardless of where the workflow / script is run.

Reply:
Interesting idea. Did you test whether it has the desired effect for the fluctuating `darkmode`?